test/bewit.js

```diff
@@ -52,16 +54,16 @@ describe('bewit scheme', function () {
         return '';
     };
 
-    var bewitHandler = function (request, reply) {
+    const bewitHandler = function (request, reply) {
 
         reply('Success');
     };
 
-    var server = new Hapi.Server();
+    let server = new Hapi.Server();
```

Comment:
This might need a little bit refactoring, the fact that a test overrides the server somewhere and the others depend on it is not very clean.